src/types.mligo

```diff
@@ -110,12 +110,12 @@ type nonce = nat
 type vote_type = bool
 
 // Voter info for a proposal
-type voter =
-  { voter_address : address
-  ; vote_amount : nat
-  ; vote_type : vote_type
+type vote_item =
+  { vote_amount : int
   }
 
+type voter_map = (address, vote_item) map
```

Comment:
I suppose this precedes our discussion and it should actually be like:
```ocaml
type vote_amount = nat
type voter =
  { voter_address : address 
  ; vote_type : vote_type
  }

type voter_map = (voter, vote_amount) map 
```

Reply:
Have changed this and have added a test to check that the tokens are unstaked correctly if there are multiple votes from a voter.